### ISSUE TITLE
Should we include instructions for building and testing the image locally?

### ISSUE THREAD

**Issue by Bucknalla** [balenablocks/fin#10]
I've been testing this by mocking the balenablocks registry using:
```bash
docker run -d -p 5000:5000 --restart=always --name registry registry:2
docker buildx build -t localhost:5000/finabler:latest --platform linux/arm/v7 --file Dockerfile.template .
docker push localhost:5000/finabler:latest 

balena deploy ${YOUR_APPLICATION}
```

Is it worth adding this to the BUILDING.md documentation?

**Comment by Bucknalla** [comment]
@phil-d-wilson thoughts on local building instructions for the block?

**Comment by phil-d-wilson** [comment]
@Bucknalla really good idea! We should do this for all the blocks, thinking about it.

**Comment by Bucknalla** [comment]
@phil-d-wilson I'll add some info to BUILDING.md :)

**Comment by Bucknalla** [comment]
Closing with fix in #15
